composer.json

```diff
@@ -23,8 +23,10 @@
         "symfony/assetic-bundle"               : "~2.6",
         "symfony/monolog-bundle"               : "~2.7",
         "symfony/swiftmailer-bundle"           : "~2.3",
-        "symfony/symfony"                      : "~2.7",
-        "twig/extensions"                      : "~1.2"
+        "symfony/symfony"                      : "~2.7.3",
+        "twig/extensions"                      : "~1.2",
+        "knplabs/knp-paginator-bundle"         : "~2.4.2",
```

Comment:
until now, the order of requirements was the following:

- php
- extensions
- packages in alphabetical order

It would make sense to put the package in the right place